smartrack-manifest.xml

```diff
@@ -18,12 +18,12 @@
   <project name="meta-freescale-distro.git" path="layers/meta-freescale-distro" remote="githf" revision="5d882cdf079b3bde0bd9869ce3ca3db411acbf3b" upstream="dunfell"/>
   <project name="meta-freescale.git" path="layers/meta-freescale" remote="githf" revision="46f54fdc5ad854a22bf759aac7fd65db1d1bb574" upstream="dunfell"/>
```

Comment:
For each of these repo, can you put a comment on what is the branch that we are pulling? Is the "revision" refers to any specific tag? If yes, will be good to list it in the comment as well. E.g. 

<!-- branch 1.4.6   tag bla-bla-bla -->

Reply:
@karhin1990 Seems like it doesn't really matter what the license file is named (as long as it is sensible). [Link](https://stackoverflow.com/questions/16627477/). Worth noting is that if we follow GitHub's guide to add a [license](https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository) to a repo, creating a file in the target repo using either "LICENSE" or "COPYING" as the filename will trigger GitHub to recognize that we are attempting to add a license. GitHub will then offer to us to choose from a license template.